execution_workflows/TAPAS/README.md

```diff
@@ -0,0 +1,54 @@
+# TAPAS
+TAPAS is an R-based software that detects polyadenylation sites within a gene from RNA-Seq data and identifies differentially polyadenylated sites between two samples.
+
+The [paper](https://academic.oup.com/bioinformatics/article/34/15/2521/4904269) is titled TAPAS: tool for alternative polyadenylation site analysis <br>
+The [application](https://github.com/arefeen/TAPAS) is free to download,
+and the [README documentation](https://github.com/arefeen/TAPAS#tapas-tool-for-alternative-polyadenylation-site-analysis) was used as a reference
+to create the nextflow pipeline of this module.
+
+This workflow qualifies for the APAeval **quantification challenge only**.
+
+## Running TAPAS workflow
+
+### Input & pre-processing
+An example sample sheet is available at `samplesheet.csv`. `samplesheet.csv` can contain multiple entries, where each row in the samplesheet has two
+columns:
+
+- sample: name of the sample for logs (e.g control_replicate1)
+- bam: BAM input file for the sample
+- read_length: read length of the sample
+
+### Docker and Singularity containers
+This workflow uses docker containers. To run, make sure that docker is installed and running
+(e.g. by running the command `docker --help` and seeing a help message printed).
+- If running with Docker, please include the `-profile docker` in the command, which enables Docker.
+- If running with Singularity, please include the `-profile singularity` in the command, which enables Singularity.
+
+### Parameters
+Parameters used to run the TAPAS are specified in the nextflow.config file.
+Parameters relevant to the workflow itself are:
+- `input` - samplesheet.csv
+
+### Running the TAPAS execution workflow
+- Download the test data [here](to be added). The current dataset is in a genomic region where there are enough reads to test TAPAS's PAS quantification functionality.
+
+- Update the samplesheet.csv with the full path to the downloaded bam, gtf, and fasta files.
+```
+sample,bam,gtf
+sample,[path_to]/SRR6795721.bam,[path_to]/gencode.v39.annotation.gtf
+```
+- Run the pipeline with the samplesheet.csv with the input paths updated using either docker or singularity containers
+```
+nextflow main.nf --input samplesheet.csv -profile <docker/singularity>
```

Comment:
I saw gtf as one of the two mandatory options in nextflow.config, does that mean we are also specifying the gtf file in the same command line here? If so we should update this command line example to include --gtf

Reply:
I have updated this with `--gtf`